src/Core/SettingsChangesHistory.cpp

```diff
@@ -250,7 +251,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
         addSettingsChanges(settings_changes_history, "24.11",
         {
             {"validate_mutation_query", false, true, "New setting to validate mutation queries by default."},
-            {"enable_job_stack_trace", false, true, "Enable by default collecting stack traces from job's scheduling."},
+            {"enable_job_stack_trace", false, false, "Enables collecting stack traces from job's scheduling. Disabled by default to avoid performance overhead."},
```

Comment:
why we do this?

Reply:
Actually I just followed this example: https://github.com/ClickHouse/ClickHouse/pull/70833/files